docs/install.md

```diff
@@ -46,6 +46,17 @@ warn --- http address - http://localhost:4873/ - verdaccio/3.0.0
 
 For more information about the CLI, please [read the cli section](cli.md).
 
+You can set the registry by using the following command.
+
+```bash
+npm set registry http://localhost:4873/
+```
+or you can pass a ``--registry`` flag when needed.
```

Comment:
Please leave a blank line between lines 53 and 54, crowdin is quite picky with code blocks that collides with text.